openslides/motions/models.py

```diff
@@ -771,6 +771,25 @@ class MotionChangeRecommendation(RESTModelMixin, models.Model):
     creation_time = models.DateTimeField(auto_now=True)
     """Time when the change recommendation was saved."""
 
+    def collides_with_other_recommendation(self):
+        recommendations = (MotionChangeRecommendation.objects
+                           .filter(motion_version=self.motion_version)
+                           .exclude(pk=self.pk))
+
+        for recommendation in recommendations:
+            if not ((self.line_from < recommendation.line_from and self.line_to <= recommendation.line_from) and
+                    not (self.line_from >= recommendation.line_to and self.line_to > recommendation.line_to)):
+                return True
+
+        return False
+
+    def save(self, *args, **kwargs):
+        if self.collides_with_other_recommendation():
+            raise WorkflowError('The recommendation collides with an existing one (line %s - %s).' %
+                                (self.line_from, self.line_to))
+
+        super(MotionChangeRecommendation, self).save(*args, **kwargs)
```

Comment:
Please remove the content of the brackets and use return: `return super().save(*args, **kwargs)`

I know that Django's save does not return anything at the moment but this may change in future and this way we show that we only want to step short before the exact save function. (No big point of course. :-)